README.md

```diff
@@ -836,6 +836,15 @@ Loadable({
 
 This option can be automated with the [Babel Plugin](#babel-plugin).
 
+#### `opts.throwOnError`
+
+Whether or not the loadable will throw when rendering if `state.error` is true.
+This option defaults to `true` so that we don't have to include error handling
+in every `loading` function.  In this situation, an error boundary should be used
+higher up in the component hierachy to catch the exception.
+
+**WARNING:** If this is set to `false`, the `loading` function must handle `props.error`.
```

Comment:
**question:** Must it? Do we enforce that? Or should this say "the `loading` function is expected to handle `props.error`?

Reply:
We don't enforce it, but it should be handling `props.error`.  If it doesn't and there is an error then the loadable will render the `loading` spinner forever.  I'll update this comment explain why proper handling of `props.error` is important in this situation.